packages/Card/src/components/Avatar/Avatar.styles.js

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components";
+import stylers from "@paprika/stylers";
+import tokens from "@paprika/tokens";
+
+export const avatarSizeStyles = {
+  small: `
+    border-radius: 10px;
+    height: 30px;
```

Comment:
width and height could use the same variable set in this file. Same for medium below